medusa/tv/series.py

```diff
@@ -1681,6 +1716,7 @@ def delete_show(self, full=False):
         # remove entire show folder
         if full:
             try:
+                _ = self.validate_location  # Let's get the exception out of the way asap.
                 log.info(u'{id}: Attempt to {action} show folder {location}',
```

Comment:
Okay, see how here self.location was included in the log. Which perviously meant, that this could raise an exception. Just from the info log.

link 1701 also self.location was used. Another place where the exception could have been raised. self.location is now (what was previously self.raw_location), i'm using self.location where it makes sense. But to keep the possibiliy the exception is raised, i put it in line 1719

Reply:
I see what you are going for now.